core-services/prow/02_config/openshift/hypershift/_prowconfig.yaml

```diff
@@ -1,6 +1,161 @@
 tide:
   queries:
-  - labels:
+  - includedBranches:
+    - community-4.6
+    - community-4.7
+    - openshift-4.10
+    - openshift-4.11
+    - openshift-4.12
+    - openshift-4.13
+    - openshift-4.14
+    - openshift-4.15
+    - openshift-4.16
+    - openshift-4.17
+    - openshift-4.18
+    - openshift-4.2
+    - openshift-4.3
+    - openshift-4.4
+    - openshift-4.5
+    - openshift-4.6
+    - openshift-4.7
+    - openshift-4.8
+    - openshift-4.9
+    - release-4.0
+    - release-4.10
+    - release-4.11
+    - release-4.12
+    - release-4.13
+    - release-4.14
+    - release-4.15
+    - release-4.16
+    - release-4.17
+    - release-4.18
+    - release-4.2
+    - release-4.3
+    - release-4.4
+    - release-4.5
+    - release-4.6
+    - release-4.7
+    - release-4.8
+    - release-4.9
+    labels:
+    - approved
+    - backport-risk-assessed
+    - cherry-pick-approved
+    - jira/valid-bug
+    - jira/valid-reference
+    - lgtm
+    missingLabels:
+    - backports/unvalidated-commits
+    - do-not-merge/hold
+    - do-not-merge/invalid-owners-file
+    - do-not-merge/needs-area
+    - do-not-merge/work-in-progress
+    - jira/invalid-bug
+    - needs-rebase
+    repos:
+    - openshift/hypershift
+  - includedBranches:
+    - openshift-4.19
+    - release-4.19
+    labels:
+    - approved
+    - jira/valid-bug
+    - jira/valid-reference
+    - lgtm
+    - staff-eng-approved
```

Comment:
I don't think we need `staff-eng-approved` on 4.19, is that in place for other repos right now?

Reply:
The other _prowconfig.yamls have the same. I think the logic is that PRs to 4.19 need to be very special, as changes generally land as fast-forwards from master or main.